### PR TITLE
Add notes and troubleshooting for system deps

### PR DESCRIPTION
**Pull request by waprin**

### CODE REVIEW

**Comment by waprin** [comment]
cc @jonparrott

File: TESTING.md [hunk 1]
```diff
@@ -14,6 +14,32 @@ Before you can run tests locally you must have:
...
+#### PortAudio on OS X
+
+If you see the error
+    
+    fatal error: 'portaudio.h' file not found
+
+Try adding the following to your ~/.pydistutils.cfg file,
+substituting in your appropriate brew Cellar directory:
+
+    include_dirs=/Users/YourUsernamen/homebrew/Cellar/portaudio/19.20140130/include/
```
Comment (jonparrott) [inline TESTING.md]: note that most users install homebrew in /usr/local.

Comment (jonparrott) [inline TESTING.md]: Their [homepage](http://brew.sh/) says /usr/local. (shruggie)

Comment (waprin) [inline TESTING.md]: ok, done.

**Comment by jonparrott** [comment]
LGTM, but @jerjou should be the decider.

File: TESTING.md [hunk 2]
```diff
@@ -14,6 +14,32 @@ Before you can run tests locally you must have:
...
+    fatal error: 'portaudio.h' file not found
+
+Try adding the following to your ~/.pydistutils.cfg file,
+substituting in your appropriate brew Cellar directory:
+
+    include_dirs=/usr/local/Cellar/portaudio/19.20140130/include/
+    library_dirs=/usr/local/YourUsername/homebrew/Cellar/portaudio/19.20140130/lib/
+
+
+
```
Comment (jerjou) [inline TESTING.md]: Getting a bit newline-happy here, huh :-)

File: TESTING.md [hunk 3]
```diff
@@ -14,6 +14,32 @@ Before you can run tests locally you must have:
...
+If you see the error
+    
+    fatal error: 'portaudio.h' file not found
+
+Try adding the following to your ~/.pydistutils.cfg file,
+substituting in your appropriate brew Cellar directory:
+
+    include_dirs=/usr/local/Cellar/portaudio/19.20140130/include/
+    library_dirs=/usr/local/YourUsername/homebrew/Cellar/portaudio/19.20140130/lib/
```
Comment (jerjou) [inline TESTING.md]: I feel like it'd be more scalable to add this to the speech/README.md, and then reference that from here?

Comment (waprin) [inline TESTING.md]: Maybe, what do you have in mind in terms of linking?

The issue is right now if someone wants to just run the whole test suite, we should make it fairly straightforward to do so. So it's not speech specific in that sense. But, we probably also want instructions in the speech samples and don't want to repeat ourselves.

Comment (jerjou) [inline TESTING.md]: I thinking something like, "If you see the error blahblahblah, see [speech/README.md#troubleshooting](speech/README.md#l=123)"

Comment (waprin) [inline TESTING.md]: ok done take look

File: TESTING.md [hunk 4]
```diff
@@ -14,6 +14,22 @@ Before you can run tests locally you must have:
...
+
+    brew install portaudio
+
+If you have trouble installing a necessary system library, try filing an issue.
+
+### Troubleshooting
+
+#### PortAudio on OS X
+
+See speech/api/README.md.
```
Comment (jerjou) [inline TESTING.md]: I'd make `speech/api/README.md` a link for easier access.

**Comment by jerjou** [comment]
LGTM after linking

**Comment by waprin** [comment]
@dpebot merge when travis passes

**Comment by waprin** [comment]
thanks @dpebot

**Pull request merged**